docs/help-center/tim-features/reset-password.mdx

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_label: Reset password
+title: Reset password
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 5
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Reset Password</h2>
+    <p class="hero__subtitle">How to reset your password if you have forgotten it or want to change it.</p>
+  </div>
+</div>
+
+## Change password
+
+To change the password, please follow [these steps](../your-account/account-settings#change-password).
+
+## Reset forgotten password
+
+:::info
+You will only be able to reset your password this way if your account is on a TI-Messenger server. If no new browser window or tab opens for login, you are most likely not on a TI-Messenger server and need to contact your administrator for resetting your password.
```

Comment:
Can't we just say: If 'reset your password' is not a clickable option under the password input field, you are not on a TIM Server and should contact your administrator to reset your password?

Reply:
I think it makes more sense to have the opening browser window or tab as indicator. This is a 100% clear. If we use the nonappearance of a button as indicator, users might search for that button in vain and be annoyed.
Can you recheck if my suggestion on this is ok for you?